inst/sql/sqlite/create_log_table.sql

```diff
@@ -1,5 +1,7 @@
 CREATE TABLE dettl_import_log (
```

Comment:
I guess at some point we might look to including flyway migrations in the package, and some system for spitting those out so that it does not need to be manually copied over into the repos that use it. It's only us atm.

Having merged the montagu-db PR, I wonder if we might change this though, and have it write a "Schema version" into the table and refuse to change anything if the schema version does not match?

Reply:
Added a ticket for this https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4370